src/viur/core/skeleton.py

```diff
@@ -1363,8 +1364,15 @@ def fromSkel(cls, kindName: str, *args: list[str]) -> t.Type[RefSkel]:
         """
         newClass = type("RefSkelFor" + kindName, (RefSkel,), {})
         fromSkel = skeletonByKind(kindName)
-        newClass.__boneMap__ = {k: v for k, v in fromSkel.__boneMap__.items() if k in args}
         newClass.kindName = kindName
+        bone_map = {}
+        for arg in args:
+            if arg[-1] == "*":  # we have a wild card prefix
+                bone_map |= {k: v for k, v in fromSkel.__boneMap__.items() if fnmatch.fnmatch(k, arg)}
+            else:
+                if bone := fromSkel.__boneMap__.get(arg):
+                    bone_map[arg] = bone
```

Comment:
> @phorward Now better ?

Quite, but it's still inconsistent, and only accepts keys ending with "*". 
```suggestion
        for arg in args:
            bone_map |= {k: fromSkel.__boneMap__[k] for k in fnmatch.filter(fromSkel.__boneMap__.keys(), arg)}
```
What do you think about this?